articles/migrations/guides/extensibility-node12.md

```diff
@@ -0,0 +1,221 @@
+
+* Free tier tenants will be migrated beginning on April xx, 2020
+* Self-serve tenants will follow through early May 2020
+* Enterprise tenants will be migrated in late May and June 2020
+
+Customers will be notified two weeks in advance of their automatic migration date, with additional periodic reminders leading up to their automatic migration date.
+
+::: warning
+All tenant automatic upgrades to Node 12 will be completed no later than June xx, 2020. Most tenant upgrades will be completed before this date. Admins for affected tenants will be notified of their exact migration date.
+:::
+
+In this document, we:
+
+* Provide recommendations on how you can ensure a smooth migration for your environment
+* Detail the specific modules affected
+
+## Summary of the migration
+
+The Webtask runtime powering the following Auth0 features utilize Node 8:
+
+* Rules
+* Hooks
+* Custom database connections
+* Custom social connections
+* Extensions
+
+If you do not use any of the extensibility features mentioned above, you are not affected by this migration. **Additionally, your tenant will automatically be upgraded to use the Node 12 runtime by June xx, 2020.** This will ensure that any future extensibility code you author will be running on a secure runtime.
+
+As part of this migration, the Auth0 development team has performed extensive testing to detect any breaking changes proactively. However, there may be behavioral changes as a result of this migration. As such, we have provided a migration switch that allows you to control the migration of your environment to the new Webtask runtime using Node 12.
+
+::: note
+Private Cloud and Managed Private Cloud (PSaaS) customers have had their tenants migrated already. No further action is necessary.
+:::
+
+### Important Dates
+
+* **2019 December 31**: [Node 8 is no longer under long-term support (LTS)](https://github.com/nodejs/Release#release-schedule)
+* **2020 April xx**: The Webtask runtime using Node 12 becomes available to Auth0 customers
+* **2020 April xx**: All official Auth0 Extensions will be updated to run on Node 12 and available for you to upgrade in the **Installed Extensions** tab of the [Extensions page](${manage_url}/#/extensions)
+* **2020 April xx**: Any tenants created after this date are already using the Node.js 12 runtime, and no action is required.
+* **2020 April xx**: Tenants with NO Extensibility code will be automatically be upgraded to use Node 12
+* **2020 April xx**: All Auth0 tenants on the Auth0 public cloud which have not already migrated to the Node.js 12 runtime, will be automatically migrated between April xx, 2020 and June xx, 2020
+* **2019 June xx**: Node 8 support in Auth0 is permanently removed
+
+## How to enable the Node 12 runtime
+
+Node 12 can be enabled through the new Extensibility panel on the [Advanced Tenant Settings](${manage_url}/#/tenant/advanced) page of the Dashboard.
+
+![Runtime toggle](/media/articles/migrations/node-runtime1.png)
```

Comment:
I think we'll need some new screenshots here (since these still show Node4/8).

Reply:
@seejee , I would appreciate your help with these, or instructions on how to get this to show. My tenants only show node 8 as an option.